### PR TITLE
Adding HTTP_HOST config item to fix #45

### LINKED ISSUE #45: [Feature Request] app.ini Item for Listen address
Currently it seems as though "0.0.0.0" is hardcoded without a way to change it.

I would like to request a config item in app.ini to be able to set the listening address to a specific IP.

Use case could be a Server with several network interfaces / ip addresses in different networks and wanting peach to only be served on one of them.

Another use case (my actual use case) would be to use a webserver like apache or nginx with reverse proxy. In this case we need to only listen on localhost.

### PR DESCRIPTION
**Pull request by Vonor**

### CODE REVIEW

File: conf/app.ini [hunk 1]
```diff
@@ -1,5 +1,6 @@
 RUN_MODE = dev
 HTTP_PORT = 5555
+HTTP_HOST = 127.0.0.1
```
Comment (Unknwon) [inline conf/app.ini]: Move this line before `HTTP_PORT`

File: pkg/setting/setting.go [hunk 2]
```diff
@@ -50,6 +50,7 @@ var (
 	AppVer   string
 	ProdMode bool
 	HTTPPort int
+	HTTPHost string
```
Comment (Unknwon) [inline pkg/setting/setting.go]: Move this line before `HTTPPort`

File: pkg/setting/setting.go [hunk 3]
```diff
@@ -127,6 +128,7 @@ func NewContext() {
 	}
 
 	HTTPPort = sec.Key("HTTP_PORT").MustInt(5555)
+	HTTPHost = sec.Key("HTTP_HOST").MustString("127.0.0.1")
```
Comment (Unknwon) [inline pkg/setting/setting.go]: Move this line before `HTTPPort`

**Comment by Vonor** [comment]
Applied your suggestions

**Comment by Unknwon** [comment]
Thank you!

**Pull request merged**